test/grammar_test.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 require_relative 'helper'
 
-class GrammarTest < Test::Unit::TestCase
+class GrammarTest < Minitest::Test
```

Comment:
Maybe we can remove the `GrammarTest` if we have a complete suite as you did.
For example, the empty multiline string, could be a file `/examples/valid/empty-multiline-strings.toml`.

What do you think?